instrumentation/aws-sdk/aws-sdk-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v2_2/AbstractAwsSdkInstrumentationModule.java

```diff
@@ -27,11 +27,6 @@ public boolean isHelperClass(String className) {
     return className.startsWith("io.opentelemetry.contrib.awsxray.");
   }
 
-  @Override
```

Comment:
@laurit could you add the label to this PR to make the tests run with indy?
I tested it locally but I just want to be safe.